pkg/runctl/run_manager.go

```diff
@@ -126,31 +141,63 @@ func (c *runManager) prepareRunNamespace(pipelineRun k8s.PipelineRun) error {
 	return nil
 }
 
-func (c *runManager) copySecrets(targetNamespace string, secretNames []string, pipelineRun k8s.PipelineRun) error {
+func (c *runManager) copyPipelinePullSecret(pipelineRun k8s.PipelineRun) (string, error) {
+	scmCloneSecretName := pipelineRun.GetSpec().JenkinsFile.Secret
+	if scmCloneSecretName == "" {
+		return "", nil
+	}
```

Comment:
Inconsistent naming: "pipelinePullSecret" vs. "scmCloneSecret"

Please decide for one term and use it consistently!